### DISCUSSION TITLE
Reading console output

### DISCUSSION THREAD

**Discussion by ArminOebb** [testcontainers/testcontainers-dotnet#594] [Q&A] [answered]
Hi! Would it be possible to implement the features, like in you Java library, to read the console output of the containers?

**Comment by HofmeisterAn** [comment, marked as answer]
Something like `WithOutputConsumer`?

https://github.com/testcontainers/testcontainers-dotnet/blob/e6c6da5db379a5c19fed95381d0c04d46ae6c1a4/tests/Testcontainers.Tests/Unit/Containers/Unix/TestcontainersContainerTest.cs#L293-L328

The equivalent of `docker logs` is not yet supported, but will be soon.

**Reply by ArminOebb** [reply]
Yes, I think that already helps me a lot, thank you. I was not aware of the Consume class.

**Reply by HofmeisterAn** [reply]
The [Quickstart](https://github.com/testcontainers/testcontainers-dotnet/wiki/Quickstart) and this https://github.com/testcontainers/testcontainers-dotnet/discussions/438 might also help.
